suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -315,17 +304,16 @@ tests:
         delete_pool: true
       desc: Test stretch Cluster mon replacement - Arbiter site
 
-# Commenting test until fix for bug : https://bugzilla.redhat.com/show_bug.cgi?id=2249962
-#  - test:
-#      name: Netsplit Scenarios data-data sites
-#      module: test_stretch_netsplit_scenarios.py
-#      polarion-id: CEPH-83574979
-#      config:
-#        pool_name: test_stretch_pool8
-#        netsplit_site: DC1
-#        tiebreaker_mon_site_name: arbiter
-#        delete_pool: true
-#      desc: Test stretch Cluster netsplit scenario between data sites
+  - test:
+      name: Netsplit Scenarios data-data sites
+      module: test_stretch_netsplit_scenarios.py
+      polarion-id: CEPH-83574979
+      config:
+        pool_name: test_stretch_pool8
+        netsplit_site: DC1
+        tiebreaker_mon_site_name: arbiter
+        delete_pool: true
+      desc: Test stretch Cluster netsplit scenario between data sites
```

Comment:
Please add a comment entry
`comments: Previously failed due to BZ-2249962`

Reply:
Added the bug details.